src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -728,15 +728,16 @@ def load_arguments(self, _):
                    help='Specify it to create new virtual machines to upgrade the scale set, rather than updating the existing virtual machines.')
 
     with self.argument_context('vmss create', arg_group='Network Balancer') as c:
-        LoadBalancerSkuName = self.get_models('LoadBalancerSkuName', resource_type=ResourceType.MGMT_NETWORK)
         c.argument('application_gateway', help='Name to use when creating a new application gateway (default) or referencing an existing one. Can also reference an existing application gateway by ID or specify "" for none.', options_list=['--app-gateway'])
         c.argument('app_gateway_capacity', help='The number of instances to use when creating a new application gateway.')
         c.argument('app_gateway_sku', help='SKU when creating a new application gateway.')
         c.argument('app_gateway_subnet_address_prefix', help='The subnet IP address prefix to use when creating a new application gateway in CIDR format.')
         c.argument('backend_pool_name', help='Name to use for the backend pool when creating a new load balancer or application gateway.')
         c.argument('backend_port', help='When creating a new load balancer, backend port to open with NAT rules (Defaults to 22 on Linux and 3389 on Windows). When creating an application gateway, the backend port to use for the backend HTTP settings.', type=int)
         c.argument('load_balancer', help='Name to use when creating a new load balancer (default) or referencing an existing one. Can also reference an existing load balancer by ID or specify "" for none.', options_list=['--load-balancer', '--lb'])
-        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2017-08-01', options_list=['--lb-sku'], arg_type=get_enum_type(LoadBalancerSkuName),
+        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2017-08-01', options_list=['--lb-sku'], arg_type=get_enum_type(['Basic', 'Standard']),
+                   help="Sku of the Load Balancer to create. Default to 'Standard' when single placement group is turned off; otherwise, default to 'Basic'. The public IP is supported to be created on edge zone only when it is 'Standard'")
+        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2021-02-01', options_list=['--lb-sku'], arg_type=get_enum_type(['Basic', 'Standard', 'Gateway']),
```

Comment:
> help="Sku of the Load Balancer to create. Default to 'Standard' when single placement group is turned off; otherwise, default to 'Basic'. The public IP is supported to be created on edge zone only when it is 'Standard'"

May I ask is this help message valid for `load_balancer_sku` with `min_api='2021-02-01'`?

Reply:
It is easy to be confused if the same parameter is defined twice. Could we add `max_api=xxx` to the first one with `min_api='2017-08-01'` to make the version control information more intuitive

Reply:
> > help="Sku of the Load Balancer to create. Default to 'Standard' when single placement group is turned off; otherwise, default to 'Basic'. The public IP is supported to be created on edge zone only when it is 'Standard'"
> 
> May I ask is this help message valid for `load_balancer_sku` with `min_api='2021-02-01'`?

Good question. I only keep the help message not changed.

Reply:
> It is easy to be confused if the same parameter is defined twice. Could we add `max_api=xxx` to the first one with `min_api='2017-08-01'` to make the version control information more intuitive

Good suggestion, I will add max_api for it.